pkg/apis/storage/types.go

```diff
@@ -294,6 +294,23 @@ type CSIDriverSpec struct {
 	// +optional
 	PodInfoOnMount *bool
 
+	// Defines if the underlying volume supports changing ownership and
+	// permission of the volume before being mounted.
+	// If set to Always, SupportsFSGroup indicates that
```

Comment:
To me, "Supports" implies a boolean rather than multiple values. Would "Mode" or "Policy" fit better here?

Reply:
I've adjusted this to be FSGroupPolicy, and updated all of the references. Let me know if this is acceptable.

Reply:
FSGroupPolicy sounds ok to me, I just hope it doesn't get confused with FSGroupChangePolicy in the pod. cc @gnufied

Reply:
if we end up merging selinux and fsGroup keps - https://github.com/kubernetes/enhancements/pull/1879 then it will be renamed anyways.